.github/scripts/release-against-rancher.sh

```diff
@@ -24,6 +24,13 @@ if [ ! -e ~/.gitconfig ]; then
     git config --global user.email fleet@suse.de
 fi
 
+# Check if version is available online
```

Comment:
This check can't be done in the github action, before calling release? I guess, because we sometimes have to run the script manually?

Reply:
I just like the workflows where validations are run first and are separate from the "doer"-scripts :)